content/en/docs/concepts/storage/storage-limits.md

```diff
@@ -78,4 +53,35 @@ Refer to the [CSI specifications](https://github.com/container-storage-interface
 
 * For volumes managed by in-tree plugins that have been migrated to a CSI driver, the maximum number of volumes will be the one reported by the CSI driver.
 
+## Custom limits
+
+You can change these limits by setting the value of the
+`KUBE_MAX_PD_VOLS` environment variable, and then starting the scheduler.
```

Comment:
I think this method is deprecated in favor of dynamic limits. @bertinatto can you confirm?

Reply:
I don't think it's deprecated as it's the only way for the user to customize the limits. This is different for CSI, as the driver might have a different mechanism to customize the value.